projects/grange-core/src/lib/interfaces.ts

```diff
@@ -140,6 +140,7 @@ export interface Resource extends BaseItem {
     '@name': string;
     '@type': string;
     '@uid': string;
+    UID: string;
```

Comment:
Guillotina resources do not have a `UID` attribute

Reply:
Plone resources does not have @uid :')

Reply:
ok, so 2 options:
- Either me make both of them optional:
```
@uid'?: string;
UID?: string;
```
and in all our code, we will have to check which value exists.
- Either in the resource service, in `get` (and any method returning a resource) we normalize the data to rename `UID` into `@uid`. It could be done by a function we provide in the main app module, and we would provide it only when using Plone as backend.

I like the 2nd option :)

Reply:
I like 2nd Option also !